pymc3/distributions/multivariate.py

```diff
@@ -503,17 +503,20 @@ def __init__(self, n, p, *args, **kwargs):
         self.mode = tt.cast(tround(self.mean), 'int32')
 
     def _random(self, n, p, size=None):
+        # Set float type to float64 for numpy.
```

Comment:
can you add a comment with a link to the `numpy` issue -- that'll make it clearer why this cast exists, and if/when we can remove it